packages/reference/infrastructure/pf_website/vars.tf

```diff
@@ -13,20 +13,4 @@ variable "vpa_enabled" {
   description = "Whether the VPA resources should be enabled"
   type        = bool
   default     = false
-}
-
-variable "algolia_app_id" {
-  description = "The Algolia App ID for the search index"
-  type        = string
-}
-
-variable "algolia_search_api_key" {
-  description = "The Algolia Search API key for the search index"
-  type        = string
-  sensitive   = true
```

Comment:
This should not be marked sensitive as you said it isn't sensitive.

Reply:
being removed as the deployment env has no effect on a built next build

Reply:
No you should still keep them as input variables. We want to be able to change this without changing the code.